packages/libs/react-ui/src/components/Select/Select.tsx

```diff
@@ -42,26 +47,29 @@ export const Select: FC<ISelectProps> = forwardRef<
     <div
       className={classNames(
         containerClass,
-        disabled ? containerClassDisabled : '',
+        selectVariants[variant],
+        disabled && containerClassDisabled,
       )}
       data-testid="kda-select"
     >
-      <div className={selectContainerClass}>
-        {Icon && (
-          <span className={iconClass}>
-            <Icon size="md" />
-          </span>
-        )}
-        <select
-          aria-label={ariaLabel}
-          ref={ref}
-          className={selectClass}
-          disabled={Boolean(disabled)}
-          {...rest}
-        >
-          {children}
-        </select>
-      </div>
+      <label>
```

Comment:
why are you wrapping this in a label?

Reply:
It's just so that clicking the (optional) icon will also highlight the select field.

Reply:
I updated it now to make the original wrapping div the label, so we don't have another depth added to it.

Reply:
oh interesting, I didn't realize it worked like that. Do you know how that effects accessibility? Also we use our `InputWrapper` to add an additional label?

Reply:
Ahh ok, I didn't realise that. Then it's a label wrapped in a label now and we don't want that 😁 I'll change it back in that case!

My change came from the comment from Maarten who said the "Click listener only listens on text." which he explained to mean that if he clicks outside of the select box, like on the icon, it won't trigger the select box. But if we wrap the whole thing in a label through the inputWrapper, that shouldn't be an issue once it's implemented in the form as the label should trigger the corresponding form item.

Reply:
No, there would not be a label wrapped around it. It would be adjacent to it and connected via htmlFor. So like 
```
<label>
<label/>
<label>
  <select/>
</label>
```

Reply:
Can you achieve this by just having the select span the entire container, adding extra padding on the left, and positioning the icon absolutely with no pointerevents?

Reply:
If the for tag on the label matches the id of the input, it works the same as wrapping it around